src/app/cli/src/graphql.ml

```diff
@@ -400,6 +403,15 @@ module Make (Commands : Coda_commands.Intf) = struct
             ; fee ~doc:"Fee amount in order to send a stake delegation"
             ; memo ~doc:"Public description of a stake delegation" ]
 
+      let filter_input ~title ~arg_name ~arg_doc =
+        obj title ~coerce:Fn.id
+          ~fields:[arg arg_name ~doc:arg_doc ~typ:(non_null string)]
```

Comment:
this should be a nullable string for `BlockFilterInput`'s `relatedTo` field.

Reply:
I was talking to @Schmavery and we thought it would be okay if BlockFilterInput is non-nullable. I can address this in a future PR.